### PR TITLE
feat(BILL-CPC-249):Add customerId to Database and Bill Object

### PR DESCRIPTION
**Pull request by jumpung**
In this pull request I added customerBillId to everything that needed it

### CODE REVIEW

**Review by Trim0500 — CHANGES_REQUESTED** [review]
Can't accept this right now

Jira:

image

There is no user story put in, I don't know what this story is supposed to accomplish. There are no subtasks defined either. We can't know what the status of the story is.

The PR naming convention isn't followed, remember the format is feat(TEAMTAG-CPC-###): Desc

Lastly and credit to Justin, but the customerBillId should be a foreign key if we are to use Auth service's sessions

File: billing-service/src/main/java/com/petclinic/billing/datalayer/Bill.java [hunk 1]
```diff
@@ -27,7 +30,9 @@
 
     public Bill(){}
 
-    public Bill(int billId, Date date, String visitType, double amount){
+    //int customerId ---> Add this to constructor maybe?
```
Comment (Trim0500) [inline billing-service/src/main/java/com/petclinic/billing/datalayer/Bill.java:33]: I think we can remove the comment out here and implement the field your considering. Do you agree?

**Review by LemireCoding — APPROVED** [review]
Comments were removed like asked. Good and ready to merge

**Review by Trim0500 — APPROVED** [review]
All changes have been addressed, I approve this message!

**Review by LemireCoding — APPROVED** [review]
Comments were removed. Everything looks good to go

**Pull request merged**